### ISSUE TITLE
BarnBridge Project Call 011

### ISSUE THREAD

**Issue by DannyDesert** [BarnBridge/BarnBridge-PM#17]
## **BarnBridge Project Call 011 Agenda**

**Meeting Date/Time:** Thursday 2021/2/18 at 15:00 UTC
**Meeting Duration:** 30 mins - 1 hour
**Breakdown:** 
- Tech/Dev: 30 mins 
- Roundtable: 30 mins

### **Tech/Dev**
- DAO
 - UI updates
 - Forum Launch
- Smart YIELD
 - Audit updates
 - Launch Dates
- Smart ALPHA

### **Operations**
- DeFi Alliance
- Bounties
- Website/Discord Community Update

### **Round Table**

**Comment by amishmanish** [comment]
Q1 for Tyler: Please ELI5 Smart yield and Smart Alpha in your words.

Q2 & 3For devs: If you can share briefly; in terms of coding and algorithm-deployment, what are the steps involved in building out the Yield and Alpha products?

Q3: How are devs looking to reduce computational steps (gas costs) for complicated on-chain derivatives? Could you share any examples.

**Comment by mike9870** [comment]
Is it possible for the team discuss the project decentralization goals? Specifically surrounding BOND and it’s products not being classified as a security.

**Comment by ObiStanKenobi** [comment]
What aspects of the products will be up to the community/DAO to decide? How will this process impact the release timelines?
